server/src/main/java/org/elasticsearch/cluster/routing/allocation/decider/FilterAllocationDecider.java

```diff
@@ -72,14 +72,14 @@
     private static final String CLUSTER_ROUTING_INCLUDE_GROUP_PREFIX = "cluster.routing.allocation.include";
     private static final String CLUSTER_ROUTING_EXCLUDE_GROUP_PREFIX = "cluster.routing.allocation.exclude";
     public static final Setting.AffixSetting<String> CLUSTER_ROUTING_REQUIRE_GROUP_SETTING =
-        Setting.prefixKeySetting(CLUSTER_ROUTING_REQUIRE_GROUP_PREFIX + ".", (key) ->
-            Setting.simpleString(key, (value, map) -> IP_VALIDATOR.accept(key, value), Property.Dynamic, Property.NodeScope));
+        Setting.prefixKeySetting(CLUSTER_ROUTING_REQUIRE_GROUP_PREFIX + ".", (key) -> Setting.simpleStringWithValidator(
```

Comment:
I think we can restore the whitespace to approximately what it was beforehand.

Reply:
Sure.

Reply:
The `simpleStringWithValidator` new API name make these lines longer than 140 characters which makes Checkstyle validation fail.

I'll restore the whitespace as soon as I'll be able restore the preferred `simpleString` API name.

Reply:
Temporary fixed in 43840f138129d15ec4322f03db33379631cb3a4b.